src/components/svg-wrapper.tsx

```diff
@@ -169,10 +169,11 @@ const SvgWrapper = () => {
             const { x, y } = getMousePosition(e);
             const { x: svgX, y: svgY } = pointerPosToSVGCoord(x, y, svgViewBoxZoom, svgViewBoxMin);
             const nodesInRectangle = findNodesInRectangle(graph.current, selectStart.x, selectStart.y, svgX, svgY);
+            const edgesInRectangle = findEdgesInRectangle(graph.current, selectStart.x, selectStart.y, svgX, svgY);
```

Comment:
How about `const edgesInRectangle = findEdgesConnectedByNodes(graph.current, nodesInRectangle)`? What's the difference here?

Reply:
How can I convert edgesInRectabgle to `Set<StnId | MiscNodeId>`?

Reply:
`Set(...)`? Performance should be similar but one helper function could be saved as `findEdgesConnectedByNodes` will give you the same result as long as I understand the new change

Reply:
I can just only make it to `Set<string>` but not `Set<StnId | MiscNodeId>`  .

Reply:
`findNodesInRectangle` should return `StnId | MiscNodeId` as nodes in the graph is this type. A cast should suffice.

Reply:
```ts
findNodesInRectangle() :...
    return new Set<StnId | MiscNodeId>(graph.filterNodes((_, attr) => inRange(sX, sY, eX, eY, attr.x, attr.y)));
```

Reply:
It said "No overload matches this call."

Reply:
Okay let me fix this in next commit

Reply:
Thanks QAQ